packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -542,6 +542,41 @@ export type GetIndexerClaimArbMerkleProofsParams =
 export type GetIndexerClaimArbMerkleProofsResponse =
   GetIndexerClaimVrtxMerkleProofsResponse;
 
+/**
+ * Blitz points
+ */
+
+export interface GetIndexerBlitzPointsParams {
+  address: string;
+}
+
+export interface GetIndexerBlitzPointsResponse {
```

Comment:
is it possible to add comments for what each of these fields are?

Reply:
Done!